src/vm/frames.h

```diff
@@ -1114,17 +1114,19 @@ class FaultingExceptionFrame : public Frame
 {
     friend class CheckAsmOffsets;
 
-#if defined(_TARGET_X86_)
+#ifndef WIN64EXCEPTIONS
+#ifdef _TARGET_X86_
     DWORD                   m_Esp;
     CalleeSavedRegisters    m_regs;
     TADDR                   m_ReturnAddress;
-#endif
-
-#ifdef WIN64EXCEPTIONS
+#else  // _TARGET_X86_
+    #error "WIN32EXCEPTIONS is supported only for X86"
```

Comment:
Again, there is nothing like WIN32EXCEPTIONS. It is only windows x86 exceptions.